orion/controllers/programs/programs_test.go

```diff
@@ -11,9 +13,55 @@ func TestExample(t *testing.T) {
   }
 }
 
-func TestCheckProgram(t *testing.T) {
-  isValid := CheckValidProgram()
-  if (isValid == false) {
-    t.Errorf("Check was incorrect, got: %t, expected: %t.", isValid, false)
-  }
+func Rand(a, b int) uint {
+	return uint(rand.Int() % (b - a) + a)
 }
+
+func TestCheckProgram(t *testing.T) {
```

Comment:
I think it would also be better to split this into 4 diff testing functions. Otherwise this one test will get huge. Each test function will test an individual component of the program. So one test for checking programid, name, grades, and description.